docs/pywbemlistener/commands.rst

```diff
@@ -0,0 +1,240 @@
+  keyword args:
+
+  * 'dt' - datetime object of the time the listener received the indication, in
+    local time. The object is timezone-aware on Python 3.6 or higher.
+  * 'dt_tzname' - timezone name of the datetime object if timezone-aware, else
+    the empty string.
+  * 'h' - Host name or IP address of the host that sent the indication
+  * 'i' - pywbem.CIMInstance object with the indication instance
+  * 'c' - CIM classname of the indication instance
+  * 'p' - Case-insensitive dictionary of the indication properties, displayed
+    as blank-separated name=value items
+
+  The default format is: "{dt} {h} {c} {p}".
+
+* Calling a Python function, via the ``--indi-call MODULE.FUNCTION`` option.
+
+  ``MODULE`` must be a module name or a dotted package name in the module search
+  path, e.g. 'mymodule' or 'mypackage.mymodule'.
+
+  The current directory is added to the front of the Python module search path,
+  if needed. Thus, the module can be a single module file in the current
+  directory, for example:
+
+  .. code-block:: text
+
+      ./mymodule.py
+
+  or a module in a package in the current directory, for example:
+
+  .. code-block:: text
+
+      ./mypackage/__init__.py
+      ./mypackage/mymodule.py
+
+  ``FUNCTION`` must be a function in that module with the following interface:
+
+  .. code-block:: text
+
+      def func(indication, host)
+
+  Parameters:
+
+  * 'indication' is a :class:`pywbem.CIMInstance` object representing the CIM
+    indication that has been received. Its 'path' attribute is None.
+
+  * 'host' is a string with the host name or IP address of the indication sender
+    (typically a WBEM server).
+
+  The return value of the function will be ignored.
+
+  Exceptions raised when importing the module cause the 'pywbemlistener run'
+  command to terminate with an error. Exceptions raised by the function when
+  it is called cause an error message to be displayed on stderr of the listener
+  process and logged, if logging is enabled.
+
+The started WBEM indication listener can log any issues it encounters, to a file.
+This is enabled by using the ``-l, --logdir DIR`` general option.
+
+See :ref:`pywbemlistener start --help` for the exact help output of the command.
+
+
+.. _`pywbemlistener stop command`:
+
+``pywbemlistener stop`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``pywbemlistener stop`` command stops a running WBEM indication listener on
+the local system.
+
+Example:
+
+.. code-block:: text
+
+    $ pywbemlistener stop lis1
+    Shut down listener lis1 running at http://localhost:25989
+
+See :ref:`pywbemlistener stop --help` for the exact help output of the command.
+
+
+.. _`pywbemlistener run command`:
+
+``pywbemlistener run`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``pywbemlistener run`` command is running the WBEM indication listener code
+in a loop that never ends. While it is possible to start this command in the
```

Comment:
The "While ... " is confusing. Should we even mention it since the easy path is pywbemlistener start

Reply:
It is an externally visible and documented command, so we need to explain it at least to the extent that people understand what it does and why not to use it directly.